### PR TITLE
feat: styles cleanup refactoring

### PR DESCRIPTION
**Pull request by frabbit**
For reviewing i suggest to compare (diff) the generated and prettified css with the current master. I tried to keep generated changes minimal.

### CODE REVIEW

**Comment by frabbit** [comment]
> Some open questions / tasks :
> 
> 1. How to choose where to put a variable? The variables in `globals/vars.scss` are no different than the ones in `globals/controls.scss` with the exception of having `$olt` prefix.
> 2. We need to change the documentation accordingly.
> 3. Do we really need so many variables ( e.g. `olt-navbar-font-size` is used only once ).

1) yes, but i wanted to have one place to make it clear in one place
2) yes, although i still want to change 1 or 2 things. It would mainly affect internal docs. External changes are sparse.
3) That's exactly the problem imho, that some definitions are scattered among so many files. There is currently no way to get a good overview what font-sizes etc. are used at all. Having this central place makes it easier to discuss things with the ui team.

File: src/core/mixins/controls.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @mixin control-placeholder-style($opacity: 1) {
-  font-size: $control-font-size;
```
Comment (drinchev) [inline src/core/mixins/controls.scss]: This results in the following change : 

image

I think would be nice to have the hard-coded values as well for browsers that don't support css custom properties.

Comment (drinchev) [reply src/core/mixins/controls.scss]: Some how this also has changed, not sure if it's related : 

image

Comment (frabbit) [reply src/core/mixins/controls.scss]: Good catch, forgot to save the file before commiting ;)

**Comment by frabbit** [comment]
> Looks good. IMHO it can be a **patch** release, however there is a small difference in the final css file :
> 
> image

Yes i know, the variable was unused before, so we can either use it or remove it completely. I thought it makes sense to use it.

**Pull request merged**